modules/ensemble/lib/widget/input/form_time.dart

```diff
@@ -101,30 +125,68 @@ class TimeState extends FormFieldWidgetState<Time> {
   }
 
   void _selectTime(BuildContext context) async {
-    final picked = await showTimePicker(
-      context: context,
-      initialTime: widget._controller.initialValue ??
-          const TimeOfDay(hour: 12, minute: 0),
-      builder: (context, child) {
-        return Theme(
+    if (widget._controller.useIOSStyleTimePicker) {
+      _showCupertinoTimePicker(context);
+    } else {
+      final picked = await showTimePicker(
+        context: context,
+        initialTime: widget._controller.value ?? TimeOfDay.now(),
+        builder: (context, child) {
+          return Theme(
             data: ThemeData(
                 colorScheme: Theme.of(context)
                     .colorScheme
                     .copyWith(onPrimary: Colors.white)),
-            child: child!);
-      },
+            child: MediaQuery(
+                data: MediaQuery.of(context).copyWith(
+                    alwaysUse24HourFormat: widget._controller.use24hFormat),
+                child: child!),
+          );
+        },
+      );
+      _updateTime(picked);
+    }
+  }
+
+  void _showCupertinoTimePicker(BuildContext context) {
+    showCupertinoModalPopup<void>(
+      context: context,
+      builder: (BuildContext context) => Container(
+        height: 216,
+        padding: const EdgeInsets.only(top: 6.0),
```

Comment:
this should come from syles

Reply:
added style properties to adjust these for now like `pickerHeight`, `pickerPadding`, and `pickerBackgroundColor` applied to iOS picker.
Did try to update to `showTimePicker` as well but it has its own multiple properties to adjust the style, to customize the material widget style, will have to add 6,7 new properties that would only affect `showTimePicker`, if you say so, then will expose them other it's having the default style and `CupertinoDatePicker` is being updated correctly.